imgui.go

```diff
@@ -638,3 +653,7 @@ func IsItemHoveredV(flags int) bool {
 func IsItemHovered() bool {
 	return IsItemHoveredV(HoveredFlagsDefault)
 }
+
+func IsKeyPressed(key int) bool {
```

Comment:
Misses documentation, please add - especially that it is about the user-key-index, not the imgui-internal one.